Client/src/api/api.jsx

```diff
@@ -1,9 +1,10 @@
 import { useState, useEffect } from 'react';
 import axios from 'axios';
-import { LOCAL_SERVER, API_ADDRESS, QUERY_TYPE } from './constants';
+import { LOCAL_SERVER, API_ADDRESS, QUERY_TYPE } from '../constants';
+import apiAuthenctication from './apiAuthentication';
 
 function getLocalData() {
-  const [apiData, setApiData] = useState('');
+  const [apiData, setApiData] = useState();
```

Comment:
It is preferred that you define respective initial state for each data type. For example for boolean flag  - false, for number - 0, etc